src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+---
+title: Towards an animation smoothness metric
+subhead: Learn about measuring animations, how to think about animation frames, and overall page smoothness.
+description: Learn about measuring animations, how to think about animation frames, and overall page smoothness.
+authors:
+  - mmocny
+date: 2021-11-03
+hero: image/eqprBhZUGfb8WYnumQ9ljAxRrA72/Q32j8dRRdKSaxYaAYiNP.jpeg
+alt: A silky smooth surface
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+---
+
+You've probably experienced pages that "stutter" or "freeze"
+during scrolling or animations. We like to say that these experiences are not
+_smooth_. To address these types of issues, the Chrome team has
+been working on adding more support to our lab tooling for animation detection,
+as well making steady improvements to the rendering pipeline diagnostics within
+Chromium.
+
+We'd like to share some recent progress, offer concrete tooling guidance,
+and discuss ideas for future animation smoothness metrics. As always, we
+would love to hear your [feedback](#feedback).
+
+This post will cover three main topics:
+
+- A quick look at animations and animation frames.
+- Our current thoughts on measuring overall animation smoothness.
+- A few practical suggestions for you to leverage in lab tooling today.
+
+## What are animations?
+
+Animations bring content to life! By making content move, especially in
+response to user interactions, animations can make an experience feel more
+natural, understandable, and fun.
+
+But poorly implemented animations, or just adding too many animations, can
+degrade the experience and make it decidedly not fun at all. We've probably all
+interacted with an interface which just added too many "helpful" transition
+effects, which actually become hostile to experience when they perform poorly.
```

Comment:
```suggestion
But poorly implemented animations, or just adding too many animations, can
degrade performance and the user experience. We've probably all
interacted with an interface with too many transitions
effects, which results in a hostile user experience that performs poorly.
```

Reply:
I think this changes the meaning slightly (and arguably we should remove the second sentence to avoid this confusion).

While it's true that unnecessary animations can be annoying, I don't think we need to comment on that in this post. It should primarily be about measuring animation smoothness.